.github/workflows/sync_authors.yml

```diff
@@ -0,0 +1,59 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+---
+name: Sync authors list
+
+on:
+  schedule:
+    #        min   hr    dom   mon   dow
+    - cron: '11    01    *     *     *'     # daily at 1.11am
+  workflow_dispatch:
```

Comment:
We have the approach that "workflow_dispatch" workflows should also be limited to a few people - for example to release the prod images, we have a workflow limited to few release managers only - we probably should do something similar here.

Reply:
This was left in because I was testing the workflow in another repo first.